doc/api/esm.md

```diff
@@ -97,7 +97,13 @@ if the nearest parent `package.json` contains `"type": "module"`.
 import './startup.js'; // Loaded as ES module because of package.json
 ```
 
-## Package Scope and File Extensions
+It is recommended that package authors always include the `"type"` field, even
```

Comment:
```suggestion
Always include the `"type"` field, even
```

Reply:
Is the intent here to improve the style (I agree “It is recommended” is awkward) or to make the recommendation a lot stronger? Saying “Always include” almost makes it sound like `"type"` is required.

Reply:
Definitely ignore the suggestion if you feel it doesn't improve things, but yes, trying to eliminate overuse (throughout all our docs) of passive voice in general and "is recommended" in particular. If you want it to be gentler, maybe this?:

```suggestion
Package authors should include the `"type"` field, even
```

But leave it as is if you think it's the best way to convey what you're trying to convey.